docs/administration/fabrics/prophecy-fabrics/connections/s3.md

```diff
@@ -16,3 +26,32 @@ To create a connection with your Amazon S3 buckets, enter the following paramete
 | Secret Access Key ([Secret required](docs/administration/secrets/secrets.md)) | Your AWS secret access key                                     |
 | Region                                                                        | AWS region where your S3 bucket is located (e.g., `us-east-1`) |
 | Bucket Name                                                                   | Name of your S3 bucket                                         |
+
+## S3 permissions
+
+When you create an S3 connection in Prophecy, access permissions are tied to the credentials you use. This means you will only see the files and folders your S3 credentials have permission to access. Any actions you perform—such as reading or writing files—are done using those credentials.
+
+To fully leverage an S3 connection in Prophecy, you need the following S3 permissions:
```

Comment:
A user needs these permissions:
- For a Bucket: List, Write
- For an object/file: Read, Write

Reference: https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html?icmpid=docs_amazons3_console